src/System.Windows.Forms/tests/IntegrationTests/MauiTests/MauiRichTextBoxTests/MauiRichTextBoxTests.cs

```diff
@@ -0,0 +1,274 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Windows.Forms.IntegrationTests.Common;
+using ReflectTools;
+using WFCTestLib.Log;
+using static Interop;
+using static Interop.User32;
+
+namespace System.Windows.Forms.IntegrationTests.MauiTests
+{
+    public class MauiRichTextBoxTests : ReflectBase
+    {
+        private const int tomLink = unchecked((int)0x80000020);
+        private const int tomHidden = unchecked((int)0x80000100);
```

Comment:
```suggestion
        private const int TomLink = unchecked((int)0x80000020);
        private const int TomHidden = unchecked((int)0x80000100);
```

What does `tom` stand for?

Reply:
> What does tom stand for?

"text object model" if I remember it right